tests/resources/p2p/test_healthcheck.py

```diff
@@ -72,7 +72,7 @@ def test_get_ready(self):
         add_new_blocks(self.manager, 5)
 
         # This will make sure the peers are synced
-        while not self.conn1.is_empty():
+        for _ in range(600):
```

Comment:
Should we use the `StopWhenSynced`?

Reply:
That would be great, but the test is not using a simulator, so for now I'd rather have this small change than a refactor to use a simulator.

Reply:
Ok. I agree to do it somewhere else.